demos/init-db.php

```diff
@@ -133,7 +133,7 @@ protected function init(): void
 
         parent::init();
 
-        $this->initPreventModification();
+        //$this->initPreventModification();
```

Comment:
⚠️ THIS MUST BE REVERTED BEFORE MERGE

We want to keep no-save beaviour for local too, as it makes the demos stateless, which significantly helps while debugging some issue.

I will check why this demo needed this uncommented to reproduce the issue, maybe we can improve this for the UserAction somehow.

Reply:
done